ui/src/containers/AppVersionsTableContainer.js

```diff
@@ -56,21 +89,23 @@ export class AppVersionsTableContainer extends React.Component {
 
   getTable = (versions = []) => {
     const renderedRows = [];
-    for (let i = 0; i < versions.length; i++) {
+    versions.map(version => {
       const tempRow = [];
-      tempRow[0] = versions[i][0];
-      tempRow[1] = versions[i][1];
-      tempRow[2] = versions[i][2];
-      if (versions[i][3].isNullOrUndefined || versions[i][3] === 'Never Launched') {
+      const versionNum = version.versionNum;
+      tempRow[0] = versionNum;
+      tempRow[1] = version.numOfAppLaunches;
+      tempRow[2] = version.currentInstalls;
+      const lastLaunched = version.lastLaunchedAt;
+      if (lastLaunched.isNullOrUndefined || lastLaunched === 'Never Launched') {
         tempRow[3] = 'Never Launched';
       } else {
-        tempRow[3] = moment(versions[i][3]).format(config.dateTimeFormat);
+        tempRow[3] = moment(lastLaunched).format(config.dateTimeFormat);
       }
-      tempRow[4] = this.createCheckbox(versions[i][0].toString(), versions[i][4]);
-      tempRow[5] = this.createTextInput(versions[i][0], versions[i][5]);
+      tempRow[4] = this.createCheckbox(versionNum, version.isDisabled);
+      tempRow[5] = this.createTextInput(versionNum, version.disabledMessage);
       renderedRows.push(tempRow);
-    }
-
+      return version;
```

Comment:
as per above, this unused return can be removed by changing implementation from `.map` to `forEach`